src/EventListener/DeserializeListener.php

```diff
@@ -46,11 +46,13 @@ public function __construct(SerializerInterface $serializer, SerializerContextBu
     public function onKernelRequest(GetResponseEvent $event)
     {
         $request = $event->getRequest();
+        $requestContent = null;
         if (
             $request->isMethodSafe(false)
             || $request->isMethod(Request::METHOD_DELETE)
             || !($attributes = RequestAttributesExtractor::extractAttributes($request))
             || !$attributes['receive']
+            || ($request->isMethod(Request::METHOD_PUT) && '' === ($requestContent = $request->getContent()))
```

Comment:
If you invert the conditions (`'' === ($requestContent = $request->getContent())` && $request->isMethod(Request::METHOD_PUT)`), you don't need the trick with `??` anymore because the variable will always be defined. WDYT?